### PR TITLE
Move CSWinRT to Toolset Dependencies in Version.Details.xml

### PR DESCRIPTION
**Pull request by kythant**
This allows us to not have an exclude list in the ConvertToVersionDetails powershell script.

### CODE REVIEW

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
   <ProductDependencies>
+    <Dependency Name="Microsoft.WindowsAppSDK.AppLicensingInternal.TransportPackage" Version="1.0.0-preview1.20211111.12">
+      <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionClosed</Uri>
+      <Sha>a3b6cd2d041c342d4a5ac2ed0c90c30e9ed08f15</Sha>
+    </Dependency>
+  </ProductDependencies>
+  <ToolsetDependencies>
     <Dependency Name="Microsoft.Windows.CsWinRT" Version="1.5.0-prerelease.220124.4">
       <Uri>https://github.com/microsoft/CsWinRT</Uri>
       <Sha>c90d2e33c650b65377a3eb046f1f449d56e2ab7d</Sha>
```
Comment (manodasanW) [inline eng/Version.Details.xml:12]: Should CsWinRT.Dependency.DotNetCoreSdk**n** be CsWinRT.Dependency.DotNetCoreSdk below?

Comment (kythant) [reply eng/Version.Details.xml:12]: oh wow that's a typo. Good catch

**Review by manodasanW — APPROVED** [review]

**Pull request merged**